btr-api/src/btr_api/models/request.py

```diff
@@ -78,13 +81,16 @@ def __init__(self, data):
         self.full_name = data['fullName']
         self.email = data['email']
         self.birthdate = data['birthdate']
-        self.business_identifier = data['businessIdentifier']
+        self.business_identifier = ",".join(data['businessIdentifiers'])
```

Comment:
This likely requires more thought then changing it, likely want to drop the index and / or change the type from string to make it more clear, maybe array? maybe sort it ie .join().sort()?